CosmWasm/wasmd
# sdk50mergemain

Please disregard for the time being. 

I just recalled I wanted to bookmark this


- Setup mergify to backport v0.40.x
- Debug simulations in CI (#1446)
- Start system tests (#1410)
- Edit config to speed up debugging (#1447)
- Bump github.com/cosmos/cosmos-sdk from 0.47.2 to 0.47.3 in /tests/system
- Bump bufbuild/buf-setup-action from 1.21.0 to 1.22.0
- Ensure history position does not conflict
- Bump seq in ibctests for failed messages as well (#1464)
- Bump bufbuild/buf-setup-action from 1.22.0 to 1.23.0
- Use large resouce class for CI system tests (#1465)
- Bump bufbuild/buf-setup-action from 1.23.0 to 1.23.1
- Remove trace param in system tests
- Add updates to ibctesting framework for MS (#1472)
- Remove check for wasm limit size in state sync (#1471)
- Bump CometBFT for security fix
- Gov authorization propagation for sub-messages (#1482)
- Deprecate alias.go (#1484)
- Upgrade to wasmvm 1.3.0-rc.0 (#1486)
- Remove dependency to wasmd in system tests
- Bump github.com/cosmos/ibc-go/v7 from 7.1.0 to 7.2.0
- Add AllDenomMetadata BankQuery (#1426)
- Add Encodings For `MsgFundCommunityPool` from Distribution (#1458)
- Bump github.com/prometheus/client_golang from 1.15.0 to 1.16.0
- Bump bufbuild/buf-setup-action from 1.23.1 to 1.24.0
- Make system tests extendable for other app binaries
- use a more stable ibc-go
- fix some small errors in accordance with what's working in ibc-go and wasmd
- temporarily disable rosetta
- add a few error checks and update some deprecated tendermint calls
